src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: gray;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.moyo__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 23px 0;
+  display: block;
+}
+
+.is-active {
+  color: var(--active-color);
+  position: relative;
+}
+
+[data-qa='hover']:hover {
```

Comment:
The selector `[data-qa='hover']:hover` is used to apply a hover effect. While this works, consider using a more descriptive class name instead of a data attribute for styling purposes. This will improve the maintainability and readability of your CSS.